partner_compassion/models/parter_segment_affinity.py

```diff
@@ -0,0 +1,135 @@
+##############################################################################
+#
+#    Copyright (C) 2021 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Jonathan Guerne <jonathan.guerne@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import api, fields, models
+from math import exp
+
+
+class ResPartnerSegmentAffinity(models.Model):
+    _name = "res.partner.segment.affinity"
+
+    partner_id = fields.Many2one("res.partner", "Partner")
+    segment_id = fields.Many2one("res.partner.segment", "Segment")
+
+    affinity = fields.Float(
+        help='affinity of the partner for this segment (in percentage).')
```

Comment:
We conventionnaly use double quotes for all strings in the code.
```suggestion
        help="affinity of the partner for this segment (in percentage).")
```